.eslintignore

```diff
@@ -1 +0,0 @@
-examples/bundle.js
```

Comment:
This file doesn't actually exist so I didn't include it in the new configuration